### PR TITLE
API-6709 - Scala 2.13 upgrade

### PR DESCRIPTION
**Pull request by peteslater-ee**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,16 +15,12 @@ import bloop.integrations.sbt.BloopDefaults
 lazy val playSettings: Seq[Setting[_]] = Seq.empty
 lazy val appName = "api-platform-test-user-frontend"
 
+scalaVersion := "2.13.8"
+
 ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+ThisBuild / semanticdbEnabled := true
+ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
```
Comment (AndySpaven) [inline build.sbt:22]: This isn't required for 2.13 I think...

Comment (peteslater-ee) [reply build.sbt:22]: Really?
Nice.

File: test/uk/gov/hmrc/testuser/common/SuppressedLogFilter.scala [hunk 2]
```diff
@@ -27,11 +26,11 @@ import ch.qos.logback.core.spi.FilterReply
 import play.api.LoggerLike
 
 class SuppressedLogFilter(val messagesContaining: String) extends Filter[ILoggingEvent] {
-  private val suppressedEntries = new mutable.MutableList[ILoggingEvent]()
+  private var suppressedEntries = List[ILoggingEvent]()
```
Comment (AndySpaven) [inline test/uk/gov/hmrc/testuser/common/SuppressedLogFilter.scala:29]: Is immutability really needed here ?

Comment (peteslater-ee) [reply test/uk/gov/hmrc/testuser/common/SuppressedLogFilter.scala:29]: MutableList has gone so I replace a mutable val with a var.
Think of anything better?

Comment (AndySpaven) [reply test/uk/gov/hmrc/testuser/common/SuppressedLogFilter.scala:29]: I think there is ArrayBuffer or ListBuffer

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**